linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/FunctionEmitter.scala

```diff
@@ -2540,6 +2889,7 @@ private[emitter] class FunctionEmitter(sjsGen: SJSGen) {
             case Double_toBits =>
               genCallHelper(VarField.doubleToBits, newLhs)
             case Double_fromBits =>
+              // TODO Don't box RuntimeLong pair
```

Comment:
Intentionally left? (fine by me, maybe even a good idea to not make this PR bigger).